tests/api/v1/samples/getWithLimit.js

```diff
@@ -15,6 +15,10 @@ const u = require('./utils');
 const getWithLimit = require('../common/getWithLimit.js').getWithLimit;
 const Sample = tu.db.Sample;
 const path = '/v1/samples';
+const supertest = require('supertest');
+const api = supertest(require('../../../../index').app);
+const constants = require('../../../../api/v1/constants');
+const expect = require('chai').expect;
```

Comment:
Looks like L18 to L21 is not required for these tests to run.

Reply:
Good catch. I had set these up to debug some errors I was getting and I forgot to remove them.